.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -119,148 +156,171 @@ Please review the [TIMELINE](https://github.com/knative/release/blob/main/TIMELI
 - [knative-sandbox/kn-plugin-admin](https://github.com/knative-sandbox/kn-plugin-admin)
+  - [ ] Nightly Job [![Nightly][kn-plugin-admin-nightly-badge]][kn-plugin-admin-nightly-page]
   - [ ] Dependencies up to date - [![releasabilty][kn-plugin-admin-release-badge]][kn-plugin-admin-release-workflow]
   - [ ] Branch Cut
   - [ ] Release
-    - Prow Release Job  [![Prow][kn-plugin-admin-prow-badge]][kn-plugin-admin-prow-job]
+    - Prow Release Job [![Prow][kn-plugin-admin-prow-badge]][kn-plugin-admin-prow-job]
     - Latest Version [![Releases][kn-plugin-admin-version-badge]][kn-plugin-admin-release-page]
   - [ ] Release Notes ([run workflow][release-note-workflow])
 - [knative-sandbox/kn-plugin-event](https://github.com/knative-sandbox/kn-plugin-event)
+  - [ ] Nightly Job [![Nightly][kn-plugin-event-nightly-badge]][kn-plugin-event-nightly-page]
   - [ ] Dependencies up to date - [![releasabilty][kn-plugin-event-release-badge]][kn-plugin-event-release-workflow]
   - [ ] Branch Cut
   - [ ] Release
-    - Prow Release Job  [![Prow][kn-plugin-event-prow-badge]][kn-plugin-event-prow-job]
+    - Prow Release Job [![Prow][kn-plugin-event-prow-badge]][kn-plugin-event-prow-job]
     - Latest Version [![Releases][kn-plugin-event-version-badge]][kn-plugin-event-release-page]
   - [ ] Release Notes ([run workflow][release-note-workflow])
 - [knative-sandbox/kn-plugin-source-kafka](https://github.com/knative-sandbox/kn-plugin-source-kafka)
+  - [ ] Nightly Job [![Nightly][kn-plugin-source-kafka-nightly-badge]][kn-plugin-source-kafka-nightly-page]
   - [ ] Dependencies up to date - [![releasabilty][kn-plugin-source-kafka-release-badge]][kn-plugin-source-kafka-release-workflow]
   - [ ] Branch Cut
   - [ ] Release
-    - Prow Release Job  [![Prow][kn-plugin-source-kafka-prow-badge]][kn-plugin-source-kafka-prow-job]
+    - Prow Release Job [![Prow][kn-plugin-source-kafka-prow-badge]][kn-plugin-source-kafka-prow-job]
     - Latest Version [![Releases][kn-plugin-source-kafka-version-badge]][kn-plugin-source-kafka-release-page]
   - [ ] Release Notes ([run workflow][release-note-workflow])
 - [knative-sandbox/kn-plugin-source-kamelet](https://github.com/knative-sandbox/kn-plugin-source-kamelet)
+  - [ ] Nightly Job [![Nightly][kn-plugin-source-kamelet-nightly-badge]][kn-plugin-source-kamelet-nightly-page]
   - [ ] Dependencies up to date - [![releasabilty][kn-plugin-source-kamelet-release-badge]][kn-plugin-source-kamelet-release-workflow]
   - [ ] Branch Cut
   - [ ] Release
-    - Prow Release Job  [![Prow][kn-plugin-source-kamelet-prow-badge]][kn-plugin-source-kamelet-prow-job]
+    - Prow Release Job [![Prow][kn-plugin-source-kamelet-prow-badge]][kn-plugin-source-kamelet-prow-job]
     - Latest Version [![Releases][kn-plugin-source-kamelet-version-badge]][kn-plugin-source-kamelet-release-page]
-  - [ ] Release Notes ([run workflow][release-note-workflow])
+- [ ] Release Notes ([run workflow][release-note-workflow])
```

Comment:
need to fix the indentation of this release note

Reply:
done